### ISSUE TITLE
Audio Notification

### ISSUE THREAD

**Issue by Galliee** [lyneca/eidoclock#20]
Hasn't been working for more than a few days now, is this intentional or accidental? I keep on missing nights because of this :( Usually have this site open in a browser 24/7 just for that reason.

Also noticed the toggles to remove things are gone as well. Interface changes?

**Comment by lyneca** [comment]
I've completely rewritten the site in React.js, and am in the process of removing client-side calls to a pretty dodgy CORS server. I'll add back some form of toggle and the audio stuff ASAP :)

**Comment by Thelvaen** [comment]
You might want to check sites authorisation if you're using Chrome, I had this issue where the "Audio" authorisation was on automatic, but I had to force it to allowed to get sound working.

**Comment by lyneca** [comment]
Yeah, I thought that was just a problem with my computer specifically but it makes sense now. I'll have a look when I can.
